src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -354,10 +354,10 @@ private class ACCT_IndividualAccounts_TEST {
         //one contact should have been created
         system.assertEquals(1,insertedContacts.size());
         //the account should have the right type
-        if (strProcessor == CAO_Constants.ONE_TO_ONE_PROCESSOR)        
```

Comment:
@lparrott Technically these can still use the `CAO_Constants` class because this test class is in the package, but either way works.

Reply:
True, but to get code coverage for the new CAO_Constants_API I had to call it in a few places.

Reply:
Very good point, though if you don't mind rather than just support "code coverage", what do you think of creating a new unit test class for `CAO_Constants_API_TEST` that has a single test method to assert each of the getter methods. Something like
```
System.AssertEquals(CAO_Constants.ONE_TO_ONE_PROCESSOR, CAO_Constants_API.ONE_TO_ONE_PROCESSOR);
```

Reply:
If I can give my $0.02, I like this better because it's an instance of us eating our own dog food. It's the same we do in the triggers, where we could call the method directly but instead go through the API one. For example in https://github.com/SalesforceFoundation/Cumulus/blob/master/src/triggers/TDTM_Account.trigger#L33.

Reply:
Agreed. I'm fine with calling `if (strProcessor == CAO_Constants_API.ONE_TO_ONE_PROCESSOR)`, though I think it would be better to have a unit test for the API class rather than relying on another test to cover that code implicitly.

Reply:
You've raised a really good point. What that comes down to though is us writing code differently to work around issues with our legacy unit test code base. Long term we want to fix that, so to me I feel like we anything new should be built with an eye towards better test classes, as opposed to following the older patterns. 

I think we can still accomplish this without negatively affecting the unit test performance by using `@IsTest(IsParallel=True)` in the Unit Test Class definition. That's something we've been doing a lot as new test classes are built, especially ones that follow newer design patterns and can run in parallel.

Reply:
@lparrott Carlos and I discussed this. We've agreed to leave your code as is (where the method references the `CAO_Constants_API` class), however we've also agreed to create a new unit test class of  `CAO_Constants_API_TEST' with `@IsTest(IsParallel=True)` that has a single method and a series of `System.AssertEquals()` to validate each of the exposed global methods in the  `CAO_Constants_API` class.